nltk/stem/ARLSTem.py

```diff
@@ -0,0 +1,370 @@
+# -*- coding: utf-8 -*-
+#
+# Natural Language Toolkit: ARLSTem Stemmer
+#
+# Copyright (C) 2001-2017 NLTK Project
+#
+# Author: Kheireddine Abainia (x-programer) <k.abainia@gmail.com>
+# Algorithms: Kheireddine Abainia <k.abainia@gmail.com>
+#			  Siham Ouamour
+#			  Halim Sayoud
+# URL: <http://nltk.org/>
+# For license information, see LICENSE.TXT
+
+
+"""
+ARLSTem Arabic Stemmer
+
+The details about the implementation of this algorithm are described in:
+
+K. Abainia, S. Ouamour and H. Sayoud, A Novel Robust Arabic Light Stemmer , Journal of 
+Experimental & Theoretical Artificial Intelligence (JETAI'17), Vol. 29, No. 3, 2017, pp. 557-573.
+
+The ARLSTem is a light Arabic stemmer that is based on removing the affixes from the word (i.e. prefixes,
+suffixes and infixes). It was evaluated and compared to several other stemmers using Paice's parameters 
+(under-stemming index, over-stemming index and stemming weight), and the results showed that ARLSTem is
+promising and producing high performances. This stemmer is not based on any dictionary and can be used 
+on-line effectively.
+
+"""
+
+from __future__ import unicode_literals
+import re
+
+from nltk.stem.api import StemmerI
+
+class ARLSTem(StemmerI):
+	"""
```

Comment:
@xprogramer would you please use PEP-8 formatting

Reply:
Thanks for the suggestion